app/controllers/DashboardController.scala

```diff
@@ -213,9 +195,9 @@ class DashboardController @Inject() (
               case (ObligationStatus.Open, true)  => Some(Overdue)
               case _                              => None
             }
-          case _ if hasAnyOpenObligation && !dueDatePassed => Some(Due)
-          case _ if hasAnyOpenObligation && dueDatePassed  => Some(Overdue)
-          case _                                           => None
+          case _ if hasAnyOpenObligation && dueDatePassed => Some(Due)
+          case _ if hasAnyOpenObligation && dueDatePassed => Some(Overdue)
```

Comment:
```suggestion
          case _ if hasAnyOpenObligation && dueDatePassed => Some(Due)
          case _ if hasAnyOpenObligation && dueDatePassed => Some(Overdue)
```
Same comment as above. The `Some(Overdue)` will never be reached since when the condition is true, the first case will be picked.